test.sh

```diff
@@ -1,10 +1,14 @@
 #!/usr/bin/env bash
+set -eux
```

Comment:
this was helpful for me to see what happens during the test runs, happy to revert it.

Reply:
Nah, leave it if it makes things better for you.  I'm ok either way. :smile: